.travis.yml

```diff
@@ -1,9 +1,6 @@
 language: go
 go:
-    - 1.2
-    - 1.3
-    - 1.4
-    - 1.5
+    - 1.7
```

Comment:
why skipping 1.8 - 1.12?

Reply:
The minimum required version for the new context based functionality is 1.7. Test building should be required only for the current minimum and the tip version, since building for versions in between wouldn't produce any new or meaningful result.

Reply:
mmm that's clever, never though about it